moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -201,33 +209,40 @@ void CollisionCheck::run(const ros::TimerEvent& timer_event)
 
   // publish message
   {
-    auto msg = moveit::util::make_shared_from_pool<std_msgs::Float64>();
-    msg->data = velocity_scale_;
-    collision_velocity_scale_pub_.publish(msg);
+    auto msg = std::make_unique<std_msgs::msg::Float64>();  // TODO(adamp): consider going back to pool allocate
+    msg.get()->data = velocity_scale_;
+    collision_velocity_scale_pub_->publish(std::move(msg));
   }
 }
 
 void CollisionCheck::printCollisionPairs(collision_detection::CollisionResult::ContactMap& contact_map)
 {
   if (!contact_map.empty())
   {
+    auto& clock = *node_->get_clock();
```

Comment:
IMO this line is a little hard to decipher, and throwing `auto&` into the mix doesn't help. Would prefer if you changed it to 

`rclcpp::Clock clock ...`

Reply:
Sounds good to me, done